apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -1528,14 +1542,16 @@ void RetryAcksAndNacks(bool ack, AckError ackError)
                         RetryTemporaryFailures(idsToRetry, acksToRetry => queue.Locked(() => queue.Requeue(acksToRetry)), Enumerable.Empty<TimedId>(), null, 600);
                     }
 
-                    if (ackError.GetPermanentExceptions() is IEnumerable<AcknowledgementException> exception && exception.Any())
-                    {
-                        // For acks/nacks, bubble up the exception.
-                        throw new AggregateException(exception);
-                    }
+                    // TODO: Uncomment this code and log the permanent errors, while adding logging. This is commented as executing this path is not required without logging.
```

Comment:
Do you mean to remove this before submitting?

Reply:
I was planning to keep it so that we don’t lose track of it but happy to change based on feedback.

Reply:
We do not leave commented code in our repos, but is there a precedence or understanding in the client libs that its okay?

Reply:
Sure. I shall remove it.  Will just leave a TODO, to keep track of it.

Reply:
No, this is not usual in the .NET library repos either.

@Rishabh-V are these instructions for us if we need to diagnose issues? Users usually will add a dependency on the library, i.e. compiled code, so this will hardly be of any use to them.

I'd like to wait if @jskeet has a better idea but I think that if anything, we just document this for ourselves somewhere (can we do that in a test for instance)? If, as the comment says, the errors are already available to the users, why would we want to log them? Users can already decide for themselves whether to log them or not, and so can we if we need to diagnose some failures.

Reply:
Hey @amanda-tarafa - Earlier we were planning to log the permanent failures, but with the updated approach errors are already available to the user, so I had commented the code and added a TODO.  In the new commit, I have removed the commented code based on feedback. Thanks. cc: @maheshgattani